test/Microsoft.AspNetCore.Razor.Language.Test/IntegrationTests/TestTagHelperDescriptors.cs

```diff
@@ -158,7 +158,7 @@ public static IEnumerable<TagHelperDescriptor> EnumTagHelperDescriptors
                                 .Name("catch-all")
                                 .PropertyName("CatchAll")
                                 .AsEnum()
-                                .TypeName(typeof(MyEnum).FullName),
+                                .TypeName($"{typeof(TestTagHelperDescriptors).FullName}.{nameof(MyEnum)}"),
```

Comment:
Could directly hard code the entire string here but felt this was cleaner and consistent with others.